commit 64ce0a0d98a7a2b7b2620ede1463448b50053d88
Author: p-tricky <phloftus@gmail.com>
Date:   Wed Jan 14 14:19:41 2015 -0500

    moded history

diff --git a/lib/history.zsh b/lib/history.zsh
index 5de71c2..add887a 100644
--- a/lib/history.zsh
+++ b/lib/history.zsh
@@ -3,8 +3,9 @@ if [ -z "$HISTFILE" ]; then
     HISTFILE=$HOME/.zsh_history
 fi
 
-HISTSIZE=10000
-SAVEHIST=10000
+HISTSIZE=100000
+SAVEHIST=100000
+
 
 # Show history
 case $HIST_STAMPS in
@@ -18,6 +19,10 @@ setopt append_history
 setopt extended_history
 setopt hist_expire_dups_first
 setopt hist_ignore_dups # ignore duplication command history list
+setopt hist_ignore_all_dups
+setopt hist_expire_dups_first
+setopt hist_find_no_dups
+setopt hist_save_no_dups
 setopt hist_ignore_space
 setopt hist_verify
 setopt inc_append_history
